specs/pvs/corla.pvs

```diff
@@ -758,20 +802,23 @@ BEGIN
 
 END roles
 
-canvas_boards: THEORY
+canvass_boards: THEORY
 BEGIN
   IMPORTING roles, election
 
-  canvas_board?: [election -> canvas_board]
-END canvas_boards
+  canvass_board?: [election -> canvass_board]
+END canvass_boards
 
 %+END roles
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
-%%% \section{Cryptography}
+%%% \subsection{Cryptography}
 
-%%% We will add additional cryptographic primitives here as required.
+% We will add additional cryptographic primitives here as required.
+% @review kiniry dmz Is it correct to say that the cryptography is used
+% (via hashing) to verify that the ballot manifests and cvrs stored in the 
+% database are actually the ones uploaded by the Counties?
```

Comment:
A first easy step is to support automatic compression. They give us a csv, we compress it (as a .zip?), upload it, process it (ideally on-the-fly) and store it. Then offer the compressed version for download.